### PR TITLE
#62 Logout button implementation

### LINKED ISSUE #62: Design and implement the logout button
### Goal

Implement a simple logout button in the navbar, that when clicked will log the user out

### Dev notes

- Add to navbar w/ bootstrap icon

### PR DESCRIPTION
**Pull request by lealakkis**
close #62

### CODE REVIEW

File: app/templates/common/_nav.html [hunk 1]
```diff
@@ -1,98 +1,100 @@
...
+{#                    <li class="nav-item display-sm">#}
+{#                        <a class="dropdown-item" href="#">#}
+{#                            <i class="bi bi-gear-wide align-top"></i>#}
+{#                            <span class="align-middle">Settings</span>#}
+{#                        </a>#}
+{#                    </li>#}
+                    <li class="nav-item display-sm">
+                        <a class="dropdown-item" href="#">
+                            <i class="bi bi-box-arrow-in-right align-top"></i>
+                            <span class="align-middle">Sign Out</span>
```
Comment (domsec) [inline app/templates/common/_nav.html]: Add the form here as well.

File: app/templates/common/_nav.html [hunk 2]
```diff
@@ -1,98 +1,100 @@
...
+                                </a>
+                            </li>
+{# TODO: Uncomment when such features are added #}
+{#                            <li>#}
+{#                                <a class="dropdown-item" href="#">#}
+{#                                    <i class="bi bi-gear-wide align-top"></i>#}
+{#                                    <span class="align-middle">Settings</span>#}
+{#                                </a>#}
+{#                            </li>#}
+{#                            <li>#}
```
Comment (domsec) [inline app/templates/common/_nav.html]: Uncomment the divider

File: app/templates/common/_nav.html [hunk 3]
```diff
@@ -1,98 +1,100 @@
...
+{#                                <hr class="dropdown-divider">#}
+{#                            </li>#}
+                            <li>
+                                <form method=post action=account/signout>
+                                    <button class="dropdown-item">
```
Comment (domsec) [inline app/templates/common/_nav.html]: Remove the button click css.

```
button.dropdown-item:focus {
outline: 0;
}
```

Should do it.

**Review by domsec — CHANGES_REQUESTED** [review]

**Review by domsec — APPROVED** [review]

File: app/templates/common/_nav.html (hunk 1 above)
Comment (davrine) [reply app/templates/common/_nav.html]: fixed in latest commit

File: app/templates/common/_nav.html (hunk 2 above)
Comment (davrine) [reply app/templates/common/_nav.html]: fixed in latest commit

File: app/templates/common/_nav.html (hunk 3 above)
Comment (davrine) [reply app/templates/common/_nav.html]: fixed in latest commit

**Review by jason-gerard — APPROVED** [review]

**Pull request merged**